R/zzz.R

```diff
@@ -82,9 +82,10 @@ is_S7_type <- function(x) {
 }
 
 check_subsettable <- function(x, allow_env = FALSE) {
-  allowed_types <- c("list", if (allow_env) "environment")
+  allowed_types <- c("list", if (allow_env) "environment",
+                     "language", "pairlist")
```

Comment:
```suggestion
  allowed_types <- c("list", "language", "pairlist", if (allow_env) "environment")
```
?

Reply:
See the `QuotedCall` class in tests for how `[[` on a `class_call` (`"language"`) is useful.